From 45945f8e72eabcd9210158191bfbeadf3ea3fa66 Mon Sep 17 00:00:00 2001
From: Andy Beverley <andy@andybev.com>
Date: Sat, 27 Dec 2014 21:26:45 +0000
Subject: [PATCH 1/6] Ensure no perllocal.pod files are created during build

Although the version of ExtUtils-MakeMaker shipped with Debian does
not create perllocal.pod files, there is a chance that a local version
will have been installed that does.

If perllocal.pod files are created, they will conflict between
multiple packages.
---
 share/rules.dh.tiny |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/share/rules.dh.tiny b/share/rules.dh.tiny
index 2d33f6a..0b86f50 100755
--- a/share/rules.dh.tiny
+++ b/share/rules.dh.tiny
@@ -2,3 +2,7 @@
 
 %:
 	dh $@
+
+override_dh_auto_configure:
+	dh_auto_configure -- NO_PERLLOCAL=1
+
-- 
1.7.10.4

